miden-lib/asm/miden/kernels/tx/memory.masm

```diff
@@ -644,6 +647,16 @@ export.set_acct_nonce
     drop movup.3 push.ACCT_ID_AND_NONCE_PTR mem_storew dropw
 end
 
+#! Returns the account procedures section offset.
+#!
+#! Stack: []
+#! Output: [section_offset]
+#!
+#! - section_offset is the new account procedures section offset start.
+export.get_account_procedures_section_offset
```

Comment:
@bobbinth would you change the naming here?

Reply:
The naming looks good here too.